### PR TITLE
(CPR-591) Publish nightly containers

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

**Comment by underscorgan** [comment]
depends on https://github.com/puppetlabs/puppet_docker_tools/pull/30

**Comment by underscorgan** [comment]
That's merged and a new gem has been pushed, so this is ready to go!

File: docker/puppetdb/Dockerfile.nightly [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      org.label-schema.vendor="Puppet" \
+      org.label-schema.url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.name="PuppetDB" \
+      org.label-schema.license="Apache-2.0" \
+      org.label-schema.version="$PUPPETDB_VERSION" \
+      org.label-schema.vcs-url="https://github.com/puppetlabs/puppetdb" \
+      org.label-schema.vcs-ref="$vcs_ref" \
+      org.label-schema.build-date="$build_date" \
+      org.label-schema.schema-version="1.0" \
+      org.label-schema.dockerfile="/Dockerfile"
```
Comment (nwolfe) [inline docker/puppetdb/Dockerfile.nightly]: Should this be `Dockerfile.nightly` so it references itself?

File: docker/puppetdb/Dockerfile.nightly [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+COPY docker-entrypoint.sh /
+RUN chmod +x /docker-entrypoint.sh
+
+EXPOSE 8080 8081
+
+ENTRYPOINT ["dumb-init", "/docker-entrypoint.sh"]
+CMD ["foreground"]
+
+COPY Dockerfile /
```
Comment (nwolfe) [inline docker/puppetdb/Dockerfile.nightly]: Same thing -- should this be `Dockerfile.nightly` so it copies itself in?

**Comment by underscorgan** [comment]
@nwolfe updated!

**Pull request merged**